BUILD.md

```diff
@@ -59,35 +59,35 @@ environment variables `CC` and `AR` to the clang binaries' names:
 
     export CC=clang AR=llvm-ar
 
-However, Fedora has no [sbt][SBT] package in its own repositories.
-You'll have to install the latest [SBT] version following its
+However, Fedora has no [sbt] package in its own repositories.
+You'll have to install the latest [sbt] version following its
 website's instructions.
 
 Now you can build Daffodil from source and the sbt and daffodil
 commands you type will be able to call the C compiler.
 
-## Ubuntu 20.04
+## Ubuntu
 
 You can use the `apt` package manager to install most of the tools
 needed to build Daffodil:
 
-    sudo apt install build-essential clang-10 clang-format-10 default-jdk git libmxml-dev
+    sudo apt install build-essential clang clang-format default-jdk git libmxml-dev
 
 If you want to use clang instead of gcc, you'll have to set your
 environment variables `CC` and `AR` to the clang binaries' names:
 
-    export CC=clang-10 AR=llvm-ar-10
+    export CC=clang AR=llvm-ar-14 # or llvm-ar-10 or whatever you have
 
-However, Ubuntu has no [sbt][SBT] package in its own repositories.
-You'll have to install the latest [SBT] version following its
+However, Ubuntu has no [sbt] package in its own repositories.
```

Comment:
This may not be true any longer. I got prompted to update sbt by the ubuntu software updater recently.

Reply:
If you check your apt sources, you'll probably find the sbt project's apt repository.  When I updated this file, I checked again and there was still no sbt package in Debian or Ubuntu apt repositories.  BTW, I found out the reason for that from sbt's Wikipedia page, which says:

> The sbt project is "bootstrapped" — it uses sbt to build itself, as many compilers also do, and it considers that dogfooding is a positive feature. To the Debian project, however, that is considered a circular dependency, that they try to minimize. As a result, sbt is not yet in Debian.